packages/server/lib/controllers/connection.controller.ts

```diff
@@ -199,10 +170,13 @@ class ConnectionController {
                 return;
             }
 
-            await connectionService.deleteConnection(connection, integration_key, info?.environmentId as number, orchestrator);
-
-            const slackNotificationService = new SlackService({ orchestratorClient: getOrchestratorClient(), logContextGetter });
-            await slackNotificationService.closeAllOpenNotifications(environment.id);
```

Comment:
this seems like a mistake, let me know

Reply:
Will check on this tomorrow -- sorry for the delay

Reply:
The intention here was that if a user disconnects their Slack they would no longer have any open notifications, however I suppose if the user has open notifications disconnects but then decides they want to reconnect their open notifications would still be valid.

Reply:
okay I see, I misread because it's called admin but it's not an admin endpoint

Reply:
Yeah, it is called `admin` because all the slack connections occur on the `admin` account, so the connection is an `adminConnection`